src/components/bottom_bar/BottomBar.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react'
+import {
+  FaAngleDown,
+  FaAngleUp,
+  FaAngleRight,
+  FaAngleLeft,
+} from 'react-icons/fa'
+import { useDocumentContext } from '../document/DocumentContext'
+
+const BottomBar = () => {
+  const [isDropdownOpen, setIsDropdownOpen] = useState(true)
+  const { pdf, activePage, prevPage, nextPage, currPage } = useDocumentContext()
+  const toggleDropdown = () => {
+    setIsDropdownOpen((prevState) => !prevState)
+  }
+
+  let pagePreviews = 7 //temp static value just for now.
```

Comment:
We will create enums.ts file and move constats like this into this file, for now it is jsut fine